pkg/kubeapiserver/options/authentication.go

```diff
@@ -310,8 +310,7 @@ func (o *BuiltInAuthenticationOptions) AddFlags(fs *pflag.FlagSet) {
 			"File containing PEM-encoded x509 RSA or ECDSA private or public keys, used to verify "+
 			"ServiceAccount tokens. The specified file can contain multiple keys, and the flag can "+
 			"be specified multiple times with different files. If unspecified, "+
-			"--tls-private-key-file is used. Must be specified when "+
-			"--service-account-signing-key is provided")
```

Comment:
@mikedanese - that's unfortunate... we don't require this to be explicit when `--service-account-signing-key` is set? falling back to `--tls-private-key-file` is asking for trouble on serving cert rotation.

Reply:
--service-account-key-file is required by #95896, so should we modify the description to follow the validation.
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubeapiserver/options/authentication.go#L205

Reply:
actually, `--service-account-key-file` is required as of 1.20 (https://github.com/kubernetes/kubernetes/pull/95896)

drop the `If unspecified, --tls-private-key-file is used.` bit